src/main/kotlin/com/simplymerlin/minigameserver/minigame/oitc/phase/GamePhase.kt

```diff
@@ -0,0 +1,180 @@
+package com.simplymerlin.minigameserver.minigame.oitc.phase
+
+import com.simplymerlin.minigameserver.core.cancel
+import com.simplymerlin.minigameserver.core.state.GameState
+import com.simplymerlin.minigameserver.minigame.oitc.OneInTheChamberGame
+import io.github.bloepiloepi.pvp.damage.CustomDamageType
+import io.github.bloepiloepi.pvp.events.FinalDamageEvent
+import io.github.bloepiloepi.pvp.events.PickupEntityEvent
+import io.github.bloepiloepi.pvp.projectile.Arrow
+import net.kyori.adventure.text.Component
+import net.kyori.adventure.text.format.NamedTextColor
+import net.kyori.adventure.text.minimessage.MiniMessage
+import net.kyori.adventure.text.minimessage.tag.resolver.Placeholder
+import net.kyori.adventure.title.Title
+import net.minestom.server.coordinate.Pos
+import net.minestom.server.entity.Player
+import net.minestom.server.event.item.ItemDropEvent
+import net.minestom.server.event.player.PlayerDeathEvent
+import net.minestom.server.event.player.PlayerRespawnEvent
+import net.minestom.server.item.ItemStack
+import net.minestom.server.item.Material
+import net.minestom.server.scoreboard.Sidebar
+import kotlin.math.min
+
+class GamePhase(private val game: OneInTheChamberGame) : GameState() {
+
+    override var time = 60*10
```

Comment:
How long is this, and is it too long?

Reply:
10 minutes? it isnt too long imo the game ends at 20 kills, if nobody is doing anything the game ends